codeblog/src/components/specific/ContatUs/ContactUsParentComponent.jsx

```diff
@@ -0,0 +1,16 @@
+
+import ContactForm from './ContactForm/ContactForm';
+import ContactContent from './ContactContent/ContactContent';
+import styles from './ContactUsParentComponent.module.css';
+function ContactUs(){
+    return (
+        <div 
+            className={styles.mainDiv}
+            id='mainDiv'>
```

Comment:
please give a better id which specifies about where it is
for example here you could have given contactUsMainDiv